### PR TITLE
Product card 5.0

### PR DESCRIPTION
**Pull request by Dimoshua**
- [DEMO LINK](https://Dimoshua.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:16]: move into separated file

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+body {
+  margin: 0;
+}
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:14]: you can move it into normalize.css

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
+@import "./blocks/variables.css";
+@import "./blocks/normalize.css";
+@import "./blocks/stars.css";
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:8]: it's not a common rules, remove it from the html selector and set only where you really need it

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+  color: var(--text-color);
+}
+
+.card__buy {
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:72]: add some transition to the button

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:76]: do not hardcode height, use paddings

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Almost there!

File: src/style.css (hunk 4 above)
Comment (Dimoshua) [reply src/style.css:72]: for example: transition: 2s;
???

**Pull request closed without merging**